docs/framework-app_dynamics_agent.md

```diff
@@ -41,6 +43,37 @@ The framework can be configured by modifying the [`config/app_dynamics_agent.yml
 | `repository_root` | The URL of the AppDynamics repository index ([details][repositories]).
 | `version` | The version of AppDynamics to use. Candidate versions can be found in [this listing][].
 
+
```

Comment:
This is unnecessary as it's in the RP and should not be documented in the repository.